.test-infra/jenkins/job_LoadTests_ParDo_Java.groovy

```diff
@@ -97,7 +97,7 @@ def commonLoadTestConfig = { jobType, isStreaming, datasetName ->
                                                       "valueSizeBytes": 90
                                                     }
                                                """.trim().replaceAll("\\s", ""),
-                            iterations          : 10,
+                            iterations          : 1,
```

Comment:
Agree that this should be done in java too. Nit before merging - while you'll be working on the branch history - could you introduce the java change in separate commit?

Reply:
Alright